lib/item_repo.rb

```diff
@@ -0,0 +1,32 @@
+require 'CSV'
+require './lib/cleaner'
+
+class ItemRepository
+  attr_reader 
+
+  def initialize(file = './data/items.csv')
+    @file = file
+    @cleaner = Cleaner.new
+    @items = []
+    # @data = CSV.open @file, headers: true, header_converters: :symbol
+    @contents = @cleaner.open_csv(@file)
+  end
+
+  def read_contents
+    File.read(@file)
+  end
+
+  def all
+    @contents.each do |row|
+      @items << row
+    end
+    @items.length
+  end
```

Comment:
So theres no excess info in the CSV? We were stressing over nothing? lol

Reply:
Yeah it seems like as long as it is technically an item, that's fine for the 'all'. We'll get rid of the junk later